app/org/sagebionetworks/bridge/models/healthdata/HealthDataUserConsent.java

```diff
@@ -0,0 +1,85 @@
+package org.sagebionetworks.bridge.models.healthdata;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import org.joda.time.DateTime;
+
+import org.sagebionetworks.bridge.json.DateTimeToStringSerializer;
+import org.sagebionetworks.bridge.json.JodaDateTimeDeserializer;
+
+/**
+ * User consent metadata stored with the health data record. We store this in the Health Data Record table, so we can
+ * avoid doing a bunch of extra calls to other tables during export.
+ */
+@JsonDeserialize(builder = HealthDataUserConsent.Builder.class)
+public class HealthDataUserConsent {
```

Comment:
After a brief discussion, we decided that it's not appropriate to put consentedOn in the user data. So I'm ripping all that out and replacing it with only the Sharing Scope (defaults to "no sharing" if not present, of course).